data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/stream/StreamAcknowledgementManager.java

```diff
@@ -131,7 +131,7 @@ private void monitorAcknowledgment(final ExecutorService executorService, final
     private void checkpoint(final String resumeToken, final long recordCount) {
         LOG.debug("Perform regular checkpointing for resume token {} at record count {}", resumeToken, recordCount);
         partitionCheckpoint.checkpoint(resumeToken, recordCount);
-        this.recordsCheckpointed.increment(recordCount);
+        this.recordsCheckpointed.increment();
```

Comment:
So this metric now means "times" checkpointed instead of "records" checkpointed?